frappe/public/js/frappe/form/grid_row.js

```diff
@@ -1191,7 +1296,8 @@ export default class GridRow {
 	}
 	hide_form() {
 		if (frappe.utils.is_xs()) {
-			$(this.grid.form_grid).css("min-width", "1000px");
+			$(this.grid.form_grid).css("min-width", "738px");
```

Comment:
Where did you get this number from?

Reply:
md breakpoint - padding ( left & right )